android_app/app/src/main/res/values/strings.xml

```diff
@@ -27,7 +27,7 @@
 
     <string name="label_weight">Weight</string>
     <string name="label_bmi">Body mass index (BMI)</string>
-    <string name="label_bmr">Basal metabolic rate (BMR)</string>
+    <string name="label_bmr">Resting metabolic rate (RMR)</string>
```

Comment:
@oliexdev: what's your view on changing BMR -> RMR?

Reply:
I think most people don't know what RMR is (me included).

I think it is better to stay with BMR. I am not an expert in medicine but the [The Mifflin St Jeor Equation](https://en.wikipedia.org/wiki/Basal_metabolic_rate) which we use listed as BMR!?

Reply:
That was my thinking as well. Even if RMR is more "correct" (not saying it is as I don't know that) there is a value in using a term that people recognize. BMR is also the term I've seen being used in scale documentation.

Reply:
Most people don't know what BRM is either. In the field of non-medical people, that know either, they will often mistake one for the other.

Fortunately it states "metabolic rate", so even without any prior knowledge, it is descriptive. Should anyone know one and want to know how it differs, there is plenty of resources to consult.

My view is to use RMR for RMR and BRM for BRM.

Reply:
RMR and BMR values differs only about 10% with *clinical* devices! 

Normally the BMR is calculated with the [Harris–Benedict_equation](https://en.wikipedia.org/wiki/Harris%E2%80%93Benedict_equation) but this formula was revised by Mifflin and St Jeor in 1990 which is the formula we use. openScale is fully transparent in the calculation we are using. 

In the end, all these formulas are estimates that are close to each other. 

Alternative we add a new RMR with the formula by Mifflin and St Jeor  and use BMR with the formula by Roza and Shizgal. But again both values are close to each other and most people don't know the different. I see here more an academic problem. 

What I don't want is to refactor all names in the source code, so I will close this PR.

Reply:
I am not saying it doesn't. Nor do I care all that much, I can change it back if you open this again.
The 44 other fixes only exist in Norwegian Bokmål right now.